bundle/manifests/node-healthcheck-operator.clusterserviceversion.yaml

```diff
@@ -223,15 +223,26 @@ spec:
     Remediation” API to instantiate the configured remediation template which
     specifies the mechanism/controller to be used.
 
-    ### Failure handling: Self Node Remediation
-    By default NHC depends on the “Self Node Remediation” (SNR) operator, which
-    is installed automatically.
+    ### Failure handling: External Remediators
+    NHC is part of the Medik8s team that has different remediators for handling node failure,
+    e.g., “Self Node Remediation” (SNR), "Fence Agents Remediation" (FAR), "Machine Deletion Remediation" (MDR)
```

Comment:
what about removing "e.g." and making this a list? Might be easier to read than multiple remdiators in a sentence